app/templates/frameworks/sign_framework_agreement.html

```diff
@@ -0,0 +1,60 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+    {{ framework.name }} Sign agreement – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+    {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Digital Marketplace",
+        "href": "/"
+      },
+      {
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
+      {
+        "text": framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
+      },
+      {
+        "text": "Sign agreement"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+
+{% block mainContent %}
+    {% include 'toolkit/forms/validation.html' %}
+    <span class="govuk-caption-xl">{{ framework.name }} {{ contract_title }}</span>
+    <h1 class="govuk-heading-xl">Supplier appointment terms</h1>
+    {% with company_details=company_details %}
+        <p class="govuk-!-font-weight-bold">
+            If any of the information about your company is incorrect contact <a href="mailto:cloud_digital@crowncommercial.gov.uk">cloud_digital@crowncommercial.gov.uk</a> before signing.
+        </p>
+        {% include "frameworks/_framework_agreement_legal_text_{0}.html".format(framework_slug) %}
+        <p class="govuk-clearfix">
+            <a href="{{ framework_pdf_url }}">
+            <img class="framework-agreement-thumbnail" alt="" src="{{ url_for('static', filename='images/pdf-thumbnails/{0}.png'.format(framework_slug)) }}">
+                Read {{ framework.name }} {{  contract_title }}
```

Comment:
This text is taken directly from  the prototype: 
![Screenshot 2020-08-25 at 17 01 36](https://user-images.githubusercontent.com/1764158/91198634-b3bae480-e6f4-11ea-82f4-866fb1e1d500.png)
CC @NoraGDS in case it is a typo?

Reply:
Good spot. It should have 'the'

Reply:
Pushed a commit to fix 👍